src/django/api/models/facility/claims_reason.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+class ClaimsReason(models.Model):
```

Comment:
Nitpick: use singular naming form for models. `ClaimReason` instead of `ClaimsReason`